packages/kit/src/runtime/client/client.js

```diff
@@ -45,6 +45,9 @@ import { HttpError, Redirect, SvelteKitError } from '../control.js';
 import { INVALIDATED_PARAM, TRAILING_SLASH_PARAM, validate_depends } from '../shared.js';
 import { get_message, get_status } from '../../utils/error.js';
 import { writable } from 'svelte/store';
+import { VERSION } from 'svelte/compiler';
```

Comment:
I'd put these two lines up with the other svelte import just to keep things a little organized